apps/block_scout_web/assets/js/lib/redux_helpers.js

```diff
@@ -2,10 +2,80 @@ import $ from 'jquery'
 import _ from 'lodash'
 import { createStore as reduxCreateStore } from 'redux'
 
+/**
+ * Create a redux store given the reducer.
+ *
+ * It also enable the Redux dev tools in the development environment.
```

Comment:
Redux dev tools is also enabled in prod, so we can ✂️ this extra information.
There's no check for environment, only if the user has `__REDUX_DEVTOOLS_EXTENSION__` installed.

```suggestion
 * It also enables Redux dev tools.
```